From: Sam Thursfield <sam@afuera.me.uk>
Date: Sat, 24 Nov 2018 11:24:22 +0100
Subject: functional-tests: Require Bash for test runner

We need to clean up on errors. There isn't a 'standard' way to do this,
so this script needs to explicitly depend on Bash.

Fixes https://gitlab.gnome.org/GNOME/tracker-miners/issues/42

(cherry picked from commit dc01e6888fbee3912b208c79dc0a68279bf5feff)
---
 tests/functional-tests/test-runner.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/functional-tests/test-runner.sh b/tests/functional-tests/test-runner.sh
index 857b26c..2c3cea6 100755
--- a/tests/functional-tests/test-runner.sh
+++ b/tests/functional-tests/test-runner.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # Test runner script for Tracker's functional tests
 
